EIPS/eip-XXXX.md

```diff
@@ -0,0 +1,177 @@
+---
+Eip: XXXX *Number to be determined by Editor*
+Title: ERC Token in Multiple Languages
+Author: Jeremy Akatsa (@jeyakatsa)
+Status: Draft
+Type: Standards Track
+Category: ERC
+Created: 1/8/2022
+---
```

Comment:
For consistency with other EIPs, the header keys should be lower case (`title` instead of `Title`.)

Before merging, you'll need to add:

 - `description`
 - `discussions-to`